src/components/StatueDetailContainer.js

```diff
@@ -1,14 +1,22 @@
 import React, { Component } from "react";
-import style from "./StatueDetailContainer.module.css";
+import "./StatueDetailContainer.module.css";
 
 export class StatueDetailContainer extends Component {
   render() {
     const { name, details } = this.props.statue;
 
     return (
-      <div className={style.container}>
-        <div>{name}</div>
-        <div>{details}</div>
+      <div className="col-lg-7 col-sm-6 mt-4">
+        <div className="row mt-2">
+          <h2 className="mb-4">{name}</h2>
+          <div
+            className="col-sm-5"
+            style={{ backgroundColor: "grey", height: "400px" }}
```

Comment:
Shame not to have this in the css file.

Reply:
the inline styling is just temp to see where the picture would go and how wide it would be. We had it in the css file at first but it wouldn't override the other colours and inline would so we put it inline for now

Reply:
nevermind, I added in a default picture instead